aten/src/ATen/native/quantized/cpu/qnnpack/src/q8conv/8x8-aarch64-neon.S

```diff
@@ -19,10 +37,11 @@
 #    const void* restrict w,
 #    uint8_t* restrict c,
 #    size_t c_stride,
+#    size_t output_channel_index,
 #    const union pytorch_qnnp_q31_requantization_params quantization_params[restrict static 1])
```

Comment:
It seems to me this should be pytorch_qnnp_conv_quantization_params not pytorch_qnnp_q31_requantization_params according to q8conv.h

Reply:
Yes. This was a typo from original one. Gonna skip fixing in these PRs.